listener/common_test.go

```diff
@@ -49,7 +50,6 @@ func assertSystemInDB(t *testing.T, inventoryID string, rhAccountID *int, report
 	assert.Equal(t, system.ReporterID, reporterID)
 
 	now := time.Now().Add(-time.Minute)
-	assert.True(t, system.FirstReported.After(now), "First reported")
```

Comment:
Yes, that's correct line to remove. Then we should definitely remove FirstReported field itfelf from SystemPlatform model. It's a struct mapped to database table. We should remove so it won't fail once the column is removed from db table.